tipg/logger.py

```diff
@@ -2,4 +2,5 @@
 
 import logging
 
-logger = logging.getLogger("tipg")
+logger = logging.getLogger("uvicorn")
+logger.setLevel(logging.DEBUG)
```

Comment:
I know this is the easiest way to get the log in uvicorn but we shouldn't do this 😅 

see https://github.com/developmentseed/titiler/issues/276#issuecomment-806250688

Reply:
It's staying in there on this branch as long as this PR is marked draft!

Reply:
🙈